src/components/TeamPics/teampics.scss

```diff
@@ -0,0 +1,117 @@
+/* HEXAGON DESIGN code taken from and edited: https://www.codesmite.com/article/how-to-create-pure-css-hexagonal-grids */
+
+body {
```

Comment:
very bad practice to style the body in a component.

Reply:
Should I change the body tag into a div, section, or article tag? Or do you mean not to have 
body {
  (all css changes)
}

Reply:
in components, you should give each html element you want to style a class name and style based on the class name.

you can still do things like this 

```CSS
.teampics {
	p {
		...
	}
	
	div {
		...
	}
}
```

Reply:
Ok. So just change the body tag. Will keep it in mind when @snakethebigboss and I do a Live Share soon.